docs/source/models/causes/neonatal/preterm_birth.rst

```diff
@@ -208,10 +237,14 @@ For simplicity, we will not include YLDs in this model.
 Validation Criteria
 +++++++++++++++++++
 
-* Preterm death count and rate in simulation should match GBD estimates.
+* Preterm deaths per live birth in simulation should match GBD estimates.
 
 * No preterm deaths for simulants with LBWSG categories for gestational ages of 37 weeks or greater.
 
+* Relative Risk of preterm with and without RDS deaths due to LBWSG should match overall neonatal mortality RR.
```

Comment:
What do you mean by this, since preterm deaths will be 0 in the TMREL for LBWSG?

Reply:
I mean just for comparing between categories with $\text{GA} < 37$ weeks